docs/attributes-registry/gen-ai.md

```diff
@@ -19,6 +19,10 @@ This document defines the attributes used to describe telemetry in the context o
 | `gen_ai.request.model`           | string   | The name of the LLM a request is being made to.                                                  | `gpt-4`                                                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `gen_ai.request.temperature`     | double   | The temperature setting for the LLM request.                                                     | `0.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `gen_ai.request.top_p`           | double   | The top_p sampling setting for the LLM request.                                                  | `1.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.stop`           | string   | Up to 4 sequences where the API will stop generating further tokens.                             | `1.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
The comment is not clear to me. First, it says "4", but for example [cohere](https://docs.cohere.com/reference/chat) allows 5 sequences. Second, the example isn't right - it's a sequence so it should be a string, not a number. Third, type should be an array of strings IMO

Reply:
Good catch! Thanks @nirga, I will fix and push these up